shared/message/skill/Look.proto

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2023 NUbots <nubots@nubots.net>
+ */
+syntax = "proto3";
+
+package message.skill;
+import "Vector.proto";
+
+/// A message telling the robot to look in the given direction
+message Look {
+    /// Vector from the camera to the point to look at, in camera space
+    vec3 rPCc = 1;
```

Comment:
I would put this in world space instead of camera space, like rPCw.
The one that does things in camera space already exists as Head IK

Reply:
Kind of HeadIK but kind of not because this module also acts as a filter for smoothing the movement.

Reply:
Well at the very least, this can't be `rPCc`, because the camera space would move as you tried to align the target. it would need to be something like `rPCt`

Reply:
I would say this is beyond the scope of the PR to fix the actual head controller implementation and instead an issue can be made. 
For anyone else reading this, for context: The current implementation doesn't work the way it is intended to work, and instead works as a derivative-style method. The input should be in torso space, but this causes issues with oscillation.